README.md

```diff
@@ -127,7 +148,7 @@ CAT_OAUTH_RESTRICT_REQUESTER_ACCESS=team:123456
 
 ## Contributing
 
-[Contributions](https://github.com/src-d/code-annotation/issues) are more than welcome, if you are interested please take a look to our [Contributing Guidelines](CONTRIBUTING.md).
+[Contributions](https://github.com/src-d/code-annotation/issues) are more than welcome, if you are interested please take a look to our [Contributing Guidelines](CONTRIBUTING.md). You have more information on how to deploy it for [development purposes here](CONTRIBUTING.md#Development).
```

Comment:
@carlosms your English is better than mine. Are you sure `deploy` is the correct word here? For developers, we give instructions how to `run locally` not `deploy`. But I can be wrong.

Reply:
`how to deploy it for ...` -> `how to build and run locally for ...` ?

Reply:
You are right, _running locally_ is more clear. Fixed in https://github.com/src-d/code-annotation/pull/177/commits/1bde8b93ddc5a87cf976dfa39b7323d902cb1704